src/libraries/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/StreamExtensions.netstandard2.0.cs

```diff
@@ -6,18 +6,16 @@
 
 namespace System.Reflection.Internal
 {
-    internal static class FileStreamReadLightUp
+    internal static partial class StreamExtensions
     {
-        private static bool IsReadFileAvailable => Path.DirectorySeparatorChar == '\\';
+        private static bool IsWindows => Path.DirectorySeparatorChar == '\\';
```

Comment:
`RuntimeInformation.IsOSPlatform(OSPlatform.Windows)`?

Reply:
That also works but I kept using `DirectorySeparatorChar` because the check is simpler (we compare two characters vs two strings), and `RuntimeInformation` would require referencing another assembly from S.R.M for just one API (I guess?).

Reply:
it's a one time cost, not sure about the implementation detail. either way, treat as optional